packages/recomponents/src/components/r-tabs/r-tabs.vue

```diff
@@ -2,20 +2,27 @@
     <div>
         <div class="r-tab" :class="[{'r-tab-divided': divided}, menuClass]" role="tablist">
             <div v-for="(tab, index) in tabs" class="r-tab-item" :key="index">
-                <a v-if="tab.to"
-                   :to="tab.to"
-                   role="tab"
-                   class="r-tab-link"
-                   @click="selectTab(tab, index)"
-                   :class="{'is-active': tab.isActive}">
+                <button v-if="tab.to"
```

Comment:
this is an interesting situation. It makes sense to change the `a` to a `button`, but it also denies users the ability to middle click/open in a new tab. I'd like to hear thoughts from others in @Rebilly/frontend

Reply:
That's really interesting indeed!
First - should the user be able to click on a tab and open it in a new.. tab? 😄Most often than not, I've seen we don't have deeplinking for tabs. Example: Settings -> Coupons -> Expired tab. It's saved in the store, but does not have it's own route, because of the `@click="selectTab`.

Second - I don't get the `:to` prop - what does it do? This is not a router-link, so neither `a` nor `button` have any relation to it. Or am I missing something pretty obvious? 🤔

Reply:
@mathewsonke Are we sure users had this ability to begin with?
I don't seem to be able to reproduce this when `a` is used 😮 
I tried on Recomm (both locally and on prod) 🤷‍♀

Reply:
@andrey-rebilly It looks like the `:to` prop should enable the History API according to this [comment](https://github.com/Rebilly/rebilly-recomponents/blob/master/packages/recomponents/src/components/r-tabs/r-tabs.vue#L44-L55):
```js
    // enable history to work for tabs
    // when going back or forward
```

(I couldn't reproduce this either)

Reply:
I would vote against that functionality:
```js

watch: {
  $route(to, from) {
      // enable history to work for tabs
      // when going back or forward
      if (to.name === from.name) {
          const routeTab = this.getRouteTab();
          // select tab if found or fallback to the
          // first one
          this.selectTab(routeTab || this.tabs[0]);
      }
  },
}

// getRouteTab
return routableTabs
  .find(tab => this.$router
    && this.$router.resolve(tab.to).route.fullPath
    && this.$router.resolve(tab.to).route.fullPath === currentPath);
```

Having this kind of logic inside tabs component, that is being used as an external library, can make you pull your hair wondering what's going on with your selection 😄 

But I agree that's for another PR. @mathewsonke what do you think about that? If you find that weird - will you create the task or someone else should?

Reply:
I did create a ticket for it yesterday, here: https://jira.ilabsinc.com/browse/GWY-14160

I see what you're saying, in that it could be very confusing for users. I think this can be mitigated by 2 things: deeplinking should be properly documented, and it should be written in such a way that if no `to` prop is entered for any tabs, the deeplinking logic never activates.

Feel free to add comments to the JIRA ticket or open a bigger discussion in Slack! This is just my opinion :)